db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -139,6 +139,9 @@ def self.decrypt(field_name, value, primary_key)
       return value unless value.include?("$encrypted$")
 
       new(secret_key, value, field_name, primary_key).decrypt
+    rescue Fernet256::InvalidToken
+      raise unless ENV["INVALID_ANSIBLE_VALUE"]
+      ENV["INVALID_ANSIBLE_VALUE"]
```

Comment:
Does this value need to be known later for fix_auth?  Can it be any value?

Words are hard.  Maybe HARDCODE_ANSIBLE_PASSWORD or something like that?

Reply:
can be any value. In fixauth, we typically choose `"bogus"`. But it need not be the same as fix_auth